lib/her/model/parse.rb

```diff
@@ -41,6 +41,17 @@ def to_params(attributes, changes={})
             end
           end
 
+          # Faraday does not serialize everything thrown at it and
```

Comment:
Let's instead make this part of the [initial assignment](https://github.com/remiprev/her/pull/479/files#diff-9f46c6432f36a5642681207c26cf15f8R35). Also fine removing the comment.

```ruby
filtered_attributes =
  attributes.dup.each_with_object({}) do |(key, value), memo|
    value = value.serializable_hash if value.is_a? ActiveModel::Serialization
    memo[key] = value
  end.symbolize_keys
```

Reply:
Although I think this should work and it does pass in the context of this branch, I'm getting 3 failures when merged with my other branches. Investigating.

Reply:
Never mind, turned out I'd resolved a conflict the wrong way.